docs/html/development/getting-started.rst

```diff
@@ -34,12 +34,16 @@ For developing pip, you need to install :pypi:`tox`. Often, you can run
 Running pip From Source Tree
 ============================
 
-To run the pip executable from your source tree during development, run pip
-from the ``src`` directory:
+To run the pip executable from your source tree during development, install pip
+locally using editable installation (inside a virtualenv).
+You can then invoke your local source tree pip normally.
 
 .. code-block:: console
 
-    $ python src/pip --version
+    $ virtualenv venv
+    $ source venv/bin/activate
+    $ (venv) python -m pip install -e .
```

Comment:
I'd drop the "(venv)" here. Further, inside an activated environment, using pip (without python -m) is basically guaranteed to work.

Reply:
Oh, but Windows... :(

Reply:
I think we should encourage users (and developers!) to use `python -m pip` instead of `pip` in any case.
Also, why drop the `(venv)` part? I think it signifies that you are doing that in a venv

Reply:
Also with python3.3, you can create a virtualenv using [venv](https://docs.python.org/3/library/venv.html) via `python -m venv venv`

Reply:
I still think my approach is the most intuitive 😞

Reply:
Sorry, I thought I'd mentioned the reasoning behind my suggestion.

The (venv) being after the $ can confuse beginners. They would probably see an error, and it'd likely be easier to remove it eagerly rather than wait for a failure. :)

Alternatively, you should add (venv) before the $ on both the Python -m invocations.

Reply:
I see I am outnumbered on this, I'll remove the `venv` part.
Thanks for the reviews @pradyunsg @deveshks :smile:

Reply:
Also add the suggestion of using `python -m venv venv` to create the virtual environment using the inbuilt `venv` module as an alternative of using the external `virtualenv` module.

Reply:
Done both suggestions :)